### ISSUE TITLE
zsh completion broken

### ISSUE THREAD

**Issue by climann** [hetznercloud/cli#177]
`hcloud completion zsh` has a leading newline.

`source <(hcloud completion zsh)` works as intended, but the newline breaks autoloading.

[zsh documentation](http://zsh.sourceforge.net/Doc/Release/Completion-System.html#Completion-Functions):
> When compinit is run, it searches all such files accessible via fpath/FPATH and reads the first line of each of them. This line should contain one of the tags described below. Files whose first line does not start with one of these tags are not considered to be part of the completion system and will not be treated specially.

The package manager [Homebrew/brew](https://github.com/Homebrew/brew) for example writes an _hcloud file and links the file into fpath/FPATH based on the output of `hcloud completion zsh`.
This is similar to `hcloud completion zsh > /usr/local/share/zsh/site-functions/_hcloud`.

The file will be ignored by zsh, because of the leading newline.
Users have to run `source <(hcloud completion zsh)` manually instead.

**Solution**

https://github.com/hetznercloud/cli/blob/5780af6bdabbc97a02637fa47121e6262c8dd4d6/cli/completion.go#L321-L323

should be

```go
func runCompletionZsh(out io.Writer, cmd *cobra.Command) error { 
 	zshInitialization :=
`#compdef hcloud 
```

**Comment by thcyron** [comment]
Fixed with #178.

Thank you for not only reporting the problem, but also providing a fix. 👍

**Comment by climann** [comment]
No problem! 😄
